everviz/everest_hooks.py

```diff
@@ -30,12 +28,17 @@ def visualize_data(api):
 
 def write_webviz_config(config, file_path):
     with open(file_path, "w") as fh:
-        yaml.dump(config, fh)
+        yaml.dump(config, fh, default_flow_style=False)
     print("Webviz config file created: {}".format(file_path))
 
 
 def webviz_config():
     return {
         "title": "Everest Optimization Report",
-        "pages": [{"title": "Front page"}, {"content": ["content text"]}],
+        "pages": [
+            {
+                "title": "Front page",
+                "content": [{"Markdown": {"markdown_file": "README.md"}}],
```

Comment:
Do we wait with updating this part? I'm fine with if of course, we just need to remember to actually have it use the new plugins as well ;)